docs/development/building-from-sources.md

```diff
@@ -99,6 +99,9 @@ To build on MacOS, you need:
 - cmake (`brew install cmake`)
 - pkg-config (`brew install pkg-config`)
 - openssl (`brew install openssl`)
+- autoconf (`brew install autoconf`)
+- automake (`brew install automake`)
+- libtool (`brew install libtool`)
```

Comment:
Maybe let's make a single command with these?
```
brew install autoconf automaker libtool
```
as it's probably a bit annoying to do these ones by one?

Reply:
Thanks, fixed!